src/zmalloc.c

```diff
@@ -199,6 +264,25 @@ void zfree(void *ptr) {
 #endif
 }
 
+/* Similar to zfree, '*usable' is set to the usable size being freed. */
+void zfree_usable(void *ptr, size_t *usable) {
```

Comment:
@oranagra Is there a particular reason `zfree_usable()` can't simply return `usable` as a return value? Might be simpler and even more efficient.

Reply:
i think it's more consistent with zmalloc_usable, etc.
not sure why it's more efficient, certainly not measurable